nixos/modules/programs/ssh.nix

```diff
@@ -199,6 +199,33 @@ in
         '';
       };
 
+      kexAlgorithms = mkOption {
+        type = types.nullOr (types.listOf types.str);
+        default = null;
+        example = [ "curve25519-sha256@libssh.org" "diffie-hellman-group-exchange-sha256" ];
+        description = ''
+          Specifies the available KEX (Key Exchange) algorithms. Multiple algorithms must be comma-separated.
```

Comment:
The "Multiple algorithms must be comma-separated" part isn't relevant for the option and can be removed. Same for the other options.

Reply:
Fixed, thanks.